openupgrade_scripts/apriori.py

```diff
@@ -49,6 +49,8 @@
     # OCA/project
     "project_mail_chatter": "project",
     "project_task_dependency": "project",
+    # OCA/timesheet
+    "sale_timesheet_order_line_sync": "sale_project",
```

Comment:
```suggestion
    "sale_timesheet_order_line_sync": "sale_timesheet",
```
As discussed above :)

Reply:
Done